include/multi_sio.h

```diff
@@ -279,3 +279,5 @@ extern u32 MultiSioRecvDataCheck(void *recvp);
 //    void *recvp  User Receive Buffer Pointer
 
 #endif // GUARD_MULTI_SIO_H
+
+extern u32 MultiSioRecvBufChange(void);
```

Comment:
Why is this declared outside the guard?

Reply:
Also, can't we just define this in the stub without declaring it in the header? It's an ASM function right?

Reply:
Oh yeah, that was by mistake!

I guess? It does seem to work.